src/shared-components/button/components/anchorLinkButton/style.ts

```diff
@@ -0,0 +1,23 @@
+import { css } from '@emotion/core';
+
+import { style as TYPOGRAPHY_STYLE } from '../../../typography';
+import { ThemeType } from '../../../../constants';
+
+/**
+ * TODO: Consolidate, fully, this specific link style with TYPOGRAPHY_STYLE.link,
+ * or otherwise figure out how to maintain meaningful distinction.
+ */
+const anchorLinkButton = (theme: ThemeType) => css`
+  ${TYPOGRAPHY_STYLE.link(theme)};
+  background: transparent;
+  border: none;
+  padding: 0;
+  text-decoration: underline;
+  text-decoration-skip-ink: none;
```

Comment:
`text-decoration-skip-ink` was added to remove the default behavior of our fonts:

<img width="150" alt="Screen Shot 2021-02-12 at 12 31 06 PM" src="https://user-images.githubusercontent.com/13544620/107807847-5eb86c80-6d2e-11eb-9ae3-a4b1d710450a.png">
<img width="150" alt="Screen Shot 2021-02-12 at 12 30 57 PM" src="https://user-images.githubusercontent.com/13544620/107807848-5eb86c80-6d2e-11eb-9f00-b32009e911d4.png">

In our designs, they do not look like this. I do not know if that is a Figma limitation or something we want to enforce, so `text-decoration-skip-ink` gives it this presentation instead:

<img width="150" alt="Screen Shot 2021-02-12 at 12 30 38 PM" src="https://user-images.githubusercontent.com/13544620/107807824-595b2200-6d2e-11eb-955b-3337af6af997.png">
<img width="150" alt="Screen Shot 2021-02-12 at 12 30 43 PM" src="https://user-images.githubusercontent.com/13544620/107807822-58c28b80-6d2e-11eb-94bd-8515a7bf21f1.png">